src/coreclr/debug/di/rsmain.cpp

```diff
@@ -1692,7 +1692,7 @@ HRESULT Cordb::CreateProcessCommon(ICorDebugRemoteTarget * pRemoteTarget,
                                     debuggingFlags
                                    );
 
-    LOG((LF_CORDB, LL_EVERYTHING, "Handle in Cordb::CreateProcess is: %.I64x\n", lpProcessInformation->hProcess));
+    LOG((LF_CORDB, LL_EVERYTHING, "Handle in Cordb::CreateProcess is: %p\n", lpProcessInformation->hProcess));
```

Comment:
Treating a `HANDLE` as a pointer feels suspect to me, but it's actually documented in [Windows Data Types (BaseTsd.h)](<https://learn.microsoft.com/en-us/windows/win32/winprog/windows-data-types> "Windows Data Types (BaseTsd.h) - Win32 apps | Microsoft Learn")

> This type is declared in WinNT.h as follows:
>
> ```C
> typedef PVOID HANDLE;
> ```

so I suppose it's okay.